cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/InMemoryAuthorizer.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2015-2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package co.cask.cdap.gateway.handlers;
+
+import co.cask.cdap.common.UnauthorizedException;
+import co.cask.cdap.proto.id.EntityId;
+import co.cask.cdap.proto.security.Action;
+import co.cask.cdap.proto.security.Principal;
+import co.cask.cdap.security.authorization.AuthorizationPlugin;
+import co.cask.cdap.security.authorization.Authorizer;
+import com.google.common.collect.HashBasedTable;
+import com.google.common.collect.Table;
+
+import java.util.HashSet;
+import java.util.Set;
+import javax.annotation.concurrent.NotThreadSafe;
+
+/**
+ * In-memory implementation of {@link AuthorizationPlugin}.
```

Comment:
it should be {@link Authorizer}
And I see that we still have the AuthorizationPlugin interface with an unused AllowAllAuthorizationPlugin. 
We don't need these interfaced anymore right ?